api/tests/csuser/test_cs_user_functions.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding utf-8 -*-
+import mock
+
+from django.test import TestCase
+from api.models.csuser import CSUser
+from api.services import SteamAPIService
+
+
+class CsUserFunctionsTesCase(TestCase):
+
+    def test__str__returns_username(self):
+        #  Act
+        user_for_test = CSUser()
+        user_for_test.steam_username = 'Pepe'
+
+        # Assert
+        self.assertEqual(str(user_for_test), user_for_test.steam_username)
+
+    def test_CSUser_init_csgo_as_none(self):
+        #  Act
+        user_for_test = CSUser()
+        # Assert
+        self.assertIsNone(user_for_test.csgo_info)
+
+    def test_user_get_steam_id_returns_steam_id_when_exists(self):
+        #  Act
+        user_for_test = CSUser()
+        user_for_test.steam_id = '123'
+        result = user_for_test.get_steam_id
+        # Assert
+        self.assertEqual(result, '123')
+
+    @mock.patch.object(SteamAPIService, 'get_steam_id_from_nick_name')
+    def test_user_get_steam_id_returns_steam_id_when_does_not_exist(self, mock_get_steam_id_from_nick_name, ):
```

Comment:
better as test name something like: "test get_steam_id calls steam API when no steam_id property does not exist"?

Reply:
Absolutely agree. I'm bad giving names to functions :|